internal/gengapic/client_init_test.go

```diff
@@ -208,3 +208,25 @@ func TestClientInit(t *testing.T) {
 		txtdiff.Diff(t, tst.tstName, g.pt.String(), filepath.Join("testdata", tst.tstName+".want"))
 	}
 }
+
+func TestGenerateDefaultAudience(t *testing.T) {
+	want := "https://foo.googleapis.com/"
+	tests := []struct {
+		name string
+		host string
+	}{
+		{name: "plain host", host: "foo.googleapis.com"},
+		{name: "host with port", host: "foo.googleapis.com:443"},
+		{name: "host with scheme", host: "https://foo.googleapis.com"},
+		{name: "host with scheme and port", host: "https://foo.googleapis.com:1234"},
+		{name: "host is a proper audience", host: want},
```

Comment:
I'd also test that a custom scheme is preserved: `snazzy://foo.example.org:534`

Reply:
Done.